specification/network/resource-manager/Microsoft.Network/stable/2023-04-01/networkWatcher.json

```diff
@@ -4777,6 +4786,15 @@
         }
       },
       "description": "List of connection states snapshots."
+    },
+    "ConnectionMonitorEndPointLocationDetails": {
```

Comment:
it should be ConnectionMonitorEndpointLocationDetails

Reply:
resolved